### PR TITLE
New text for about page

### PR DESCRIPTION
**Pull request by jnsya**
Issue Here: https://github.com/rtcharity/eahub.org/issues/979

I had a couple of question/comments:
- Github doesn't reliably expose the real name of contributors, just their username, so that's what I've put in the technical contributors list. (See for yourself by making a curl request of contributors (`curl https://api.github.com/repos/rtcharity/eahub.org/contributors` or to a particular contributor `curl https://api.github.com/users/sebgrebe`). If you have list of the real names and want to add them instead then let me know.
- The [link to the FAQs](https://docs.google.com/document/d/1Lj2lXidJOlha-rKjtZgzpnkOnb90GNDOT0NsfXAUTWI/edit#) is just a google doc - is this the actual link, or just a temporary placeholder?

Here's how it looks on desktop:
image
image
image
image

### CODE REVIEW

**Comment by sebgrebe** [comment]
@jnsya, the real names issue is a good question. Initially, I thought we would want to put real names, but one might argue that someone could contribute under their github alias, without having to reveal their identity. That's how open source projects work.
So I think I'm changing my mind and would prefer github usernames.

Also, please add "Anastasiia Kuvshinova". She contributed code, but it got lost on github. I don't have her github account name. @victor-yunenko, do you have it? And are you OK with using github usernames?

**Review by sebgrebe — CHANGES_REQUESTED** [review]
Minor: Please convert png to jpg, as jpg is smaller in size.
@vagarwalla could you have a final look please whether you're happy with the text?

File: eahub/templates/eahub/about.html [hunk 1]
```diff
@@ -14,83 +14,129 @@
...
+
+        <p>This website is for you – its users! – and we are always looking to improve, so don’t hesitate to make suggestions, vote on new features and enhancements, or report bugs on our <a href="https://feedback.eahub.org/">feedback page</a>. Check out our <a href="https://docs.google.com/document/d/1Lj2lXidJOlha-rKjtZgzpnkOnb90GNDOT0NsfXAUTWI/edit#">FAQs</a> if you have any questions.</p>
+
+        <h4>For community builders</h4>
+        <p>We are working with a number of group organisers to help launch local group directories here on the Hub. If you’re interested in onboarding your group, we can help with mass uploading data (if you have the consent of your members), and we will consider feature requests to improve the directory for your local group.</p>
+
+        <h4>For organisations</h4>
+        <p>You can use the hub to search for and contact potential candidates for paid or voluntary roles.</p>
+        <p>If you are an EA-aligned organisation and believe you could benefit from our data, please <a href="mailto:vaidehi@eahub.org"> contact Vaidehi, our Product Manager</a>.</p>
```
Comment (sebgrebe) [inline eahub/templates/eahub/about.html:36]: I'd only put the link on "Vaidehi"

File: eahub/templates/eahub/about.html [hunk 2]
```diff
@@ -14,83 +14,129 @@
...
+        </p>
+
+        <p><strong>Technical Contributors</strong></p>
         <ul>
-            <li>Piotr Cieszynski: Tester</li>
-            <li>Alexander Herwix: Advisor</li>
-            <li>Richenda Herzig: Product Manager and Illustrator</li>
-            <li>Michal Trzesimiech: Project Manager</li>
-            <li>Nadia Williams: Illustrator</li>
+          <li><a href="https://github.com/sebgrebe">sebgrebe</a></li>
```
Comment (sebgrebe) [inline eahub/templates/eahub/about.html:69]: please put in alphabetical order

File: eahub/templates/eahub/about.html [hunk 3]
```diff
@@ -14,83 +14,129 @@
...
+          <li><a href="https://eahub.org/profile/balazs-rapi/">Balázs Rapi</a>: Operations</li>
+          <li>Haneesha Puruchuri</li>
         </ul>
+        <p>We are also grateful for the feedback of several volunteers and advisors.</p>
 
-        Data Privacy Officer: Victor Yunenko
 
-        <h4 class="mt-4"> Feedback </h4>
+        <h2>History</h2>
+        <p>The EA Hub was originally created by Tom Ash in 2014 to host both his own projects (beginning with the <a href="https://www.rethinkprojects.org/easurvey">EA Survey</a> and the public <a href="https://eahub.org/profiles/">EA Profiles</a> that came out of it) and other independent ones. Tom's projects later moved to their own non-profit, .impact Focus Projects, which was <a href="https://forum.effectivealtruism.org/posts/a3sScjxdgNTCseBMJ/impact-is-now-rethink-charity">renamed Rethink Charity</a> in 2017. The Hub is now fiscally sponsored by <a href="https://rtcharity.org/">Rethink Charity</a>, and run by our team of volunteers. </p>
```
Comment (sebgrebe) [inline eahub/templates/eahub/about.html:98]: Put "Rethink Charity" in quotes in "Renamed Rethink Charity"

**Review by sebgrebe — CHANGES_REQUESTED** [review]

**Comment by sebgrebe** [comment]
@jnsya, discussed the name issue with @victor-yunenko. We agreed on using clear names where we have them and the github usernames where not

File: eahub/templates/eahub/about.html [hunk 4]
```diff
@@ -14,83 +14,129 @@
...
+          <li><a href="https://github.com/cephalin">cephalin</a></li>
+          <li><a href="https://github.com/patbl">patbl</a></li>
+          <li><a href="https://github.com/david-mears">david-mears</a></li>
+          <li><a href="https://github.com/mattg95">mattg95</a></li>
+          <li><a href="https://github.com/LisaJD">LisaJD</a></li>
+          <li><a href="https://github.com/marcin-wosinek">marcin-wosinek</a></li>
+          <li><a href="https://github.com/berndverst">berndverst</a></li>
+          <li><a href="https://github.com/Richenda">Richenda</a></li>
+          <li><a href="https://github.com/austin226">austin226</a></li>
+          <li><a href="https://github.com/cafelow">cafelow</a></li>
```
Comment (viktor-yunenko) [inline eahub/templates/eahub/about.html:83]: could help with adding `rel=nofollow` for SEO purposes?

**Comment by viktor-yunenko** [comment]
And the developer who has been helping with the frontend - Anastasiia Kuvshinova

File: eahub/templates/eahub/about.html [hunk 5]
```diff
@@ -14,83 +14,129 @@
...
+          <li>Taymon Beal: Tech Lead/Back-End Developer</li>
+          <li>Sebastian Becker: Front-End Developer</li>
+          <li>Nadia Williams: Illustrator</li>
+          <li>Piotr Cieszynski: Tester</li>
+          <li>Alexander Herwix: Advisor</li>
+        </ul>
+
+        <h2>Get involved</h2>
+        <h4>Volunteer</h4>
+        <p>We are always <a href="http://bit.ly/eahub-openroles">looking for volunteers</a> to help build the Hub, in both technical and non-technical roles. If you’re interested in tech roles, please <a href="https://docs.google.com/forms/d/1DH5mNALNNe79Kbx8akGi56TpvrtaepAEaKsu96jZ-Ks/edit">sign up</a> before starting to work on an issue in our Repo so that we can direct you to the most pressing or suitable tasks.</p>
```
Comment (viktor-yunenko) [inline eahub/templates/eahub/about.html:114]: And well rel=nofollow everywhere else :)

Probably expect the forum and rethink charity, as they don't add it for our links

**Comment by sebgrebe** [comment]
Closing this in lieu of [this PR](https://github.com/rtcharity/eahub.org/pull/1218) as Vaidehi asked to release ASAP and there were some changes that need to be made prior to release.

**Comment by vagarwalla** [comment]
FWIW
- Text was looking good
- I'll let the tech team decide whether you want real names or not. I think EA folks may want real names? Perhaps you can send a messaging letting them know about the About page, but and if they'd prefer github username to do that.

**Pull request closed without merging**